base_layer/contacts/src/chat_client/src/client.rs

```diff
@@ -194,6 +195,18 @@ impl ChatClient for Client {
         message.push(metadata);
         message
     }
+
+    async fn get_conversationalists(&self) -> Vec<TariAddress> {
+        let mut addresses = vec![];
+        if let Some(mut contacts_service) = self.contacts.clone() {
+            addresses = contacts_service
+                .get_conversationalists()
+                .await
+                .expect("Addresses from conversations not fetched");
```

Comment:
Please change this to either return an error, or return an empty vec on error.

Reply:
I didn't handle the error here specifically because every function in this client needs to have its errors managed. I can change just this one if you want, but the last task for chat is to clean up this class to remove panic potential, and anywhere else through the contacts service. Which I planned to do in it's own PR.